ui/app/serializers/clients/activity.js

```diff
@@ -88,6 +90,14 @@ export default class ActivitySerializer extends ApplicationSerializer {
     return object;
   }
 
+  // TODO CMB make these two functions utils?
+
+  // format ISO timestamp to M/yy
+  formatTimestamp(timestamp) {
```

Comment:
Timestamp looks like: `2021-03-21T00:00:00Z` - if you directly format this, it returns a date object associated with the user's timezone. We want to globally return the same year and month as the API, so by splitting on `T` and parsing _after_ we consistently get `3/21`